easybuild/easyconfigs/o/OpenSSL/OpenSSL-1.0.1p-foss-2015b.eb

```diff
@@ -12,8 +12,6 @@ toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
 sources = [SOURCELOWER_TAR_GZ]
 source_urls = ['http://www.openssl.org/source/']
 
-patches = ['OpenSSL-%(version)s_icc-fixes.patch']
-
```

Comment:
only needed for `icc`, which is not used in the `foss` toolchain, so shouldn't be there (don't fix it if it's not broken)